src/main/resources/config/application.yml

```diff
@@ -217,6 +217,7 @@ user-registration:
 
 #Solr Configurations
 base-solr-url: http://localhost:8983/solr
+base-solr-url-8985: http://localhost:8985/solr
```

Comment:
don't include port number in variable name

Reply:
Ok, will make this change